test/e2e/test_helpers.go

```diff
@@ -279,6 +279,8 @@ func (t testHelper) deployClusterAndWait(params deployClusterParams, clusterReso
 		ControlPlaneMachineCount: pointer.Int64Ptr(1),
 		WorkerMachineCount:       pointer.Int64Ptr(1),
 	}
+	fmt.Println("apparently Custer config")
```

Comment:
are these logs useful? also nit: spelling of cluster is wrong.

Reply:
nope; just debug logs. will be cleaned up.